arm-customer-insights/2017-01-01/examples/HubsListByResourceGroup.json

```diff
@@ -0,0 +1,55 @@
+{
+	"parameters": {
+		"api-version": "2016-01-01",
+		"subscriptionId": "subid",
+		"resourceGroupName": "TestHubRG",
```

Comment:
please remove the trailing comma after `"TestHubRG",`. In JSON, there is no comma after the last item